src/services/edusharing/services/EduSharingConnector.js

```diff
@@ -44,7 +44,7 @@ const edusharingInstancePermissions = [
 		name: 'Thuringia',
 		alias: 'thr',
 		privateGroups: ['GROUP_Thuringia-private'],
-		publicGroups: ['GROUP_Thuringia-public'],
+		publicGroups: ['GROUP_public', 'GROUP_Thuringia-public'],
```

Comment:
Thuringia user belongs to GROUP_Thuringia-public. Inside Edu-Sharing WLO content gives access to GROUP_Thuringia-public. However, the elements themselves have only the initial group access value 'GROUP_public'. That's why this change was needed.